pkg/adapter/network.go

```diff
@@ -0,0 +1,122 @@
+// +build !remoteclient
+
+package adapter
+
+import (
+	"encoding/json"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+	"text/tabwriter"
+
+	"github.com/containernetworking/cni/libcni"
+	"github.com/containers/libpod/cmd/podman/cliconfig"
+	"github.com/containers/libpod/pkg/network"
+	"github.com/pkg/errors"
+)
+
+// NetworkList displays summary information about CNI networks
+func (r *LocalRuntime) NetworkList(cli *cliconfig.NetworkListValues) error {
+	networks, err := network.LoadCNIConfsFromDir(network.CNIConfigDir)
```

Comment:
Isn't the config location defined in `libpod.conf` as 
```
# Directory containing CNI plugin configuration files
cni_config_dir = "/etc/cni/net.d/"
```

Reply:
yes ... but ... im not sure we want to load in pkg/ so I was hoping to wire up the config later.